src/hooks/useMultiPhaseKycFlow.ts

```
@@ -1,20 +1,60 @@
 import { useState, useCallback, useRef, useEffect, useMemo } from 'react'
 import { useAuth } from '@/context/authContext'
 import { useSumsubKycFlow } from '@/hooks/useSumsubKycFlow'
-import { useRailStatusTracking } from '@/hooks/useRailStatusTracking'
+import { useCapabilities } from '@/hooks/useCapabilities'
+import { deriveBridgeGate } from '@/utils/bridge-gate.utils'
 import { getBridgeTosLink, confirmBridgeTos } from '@/app/actions/users'
-import { type KycModalPhase } from '@/interfaces'
+import { type KycModalPhase, type IUserProfile } from '@/interfaces'
```

ESLint

[error] '`@/interfaces`' import is restricted from being used. Import from a specific file instead of the '`@/interfaces`' barrel — barrels force the bundler to load every re-export and hurt build perf. See CLAUDE.md.

(no-restricted-imports)